File: web-ui/src/app/model/lakehouse-stats.ts
export class LakehouseStats {
    totalIndices: number;
    totalContexts: number;
    totalFiles: number;
    totalStoredFileSizeMB: number;


    constructor(totalIndices: number, totalContexts: number, totalFiles: number, totalStoredFileSizeBytes: number) {
        this.totalIndices = totalIndices;
        this.totalContexts = totalContexts;
        this.totalFiles = totalFiles;
        this.totalStoredFileSizeMB = totalStoredFileSizeBytes / 1024 / 1024;
    }
}

File: svc-shared/src/main/java/at/jku/dke/bigkgolap/shared/model/LakehouseStats.java
package at.jku.dke.bigkgolap.shared.model;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class LakehouseStats {
    private final long totalIndices;
    private final long totalContexts;
    private final long totalFiles;
    private final long totalStoredFileSizeBytes;
}
